andela/metis-ah-frontend
# #159987369 Pagination support for articles

#### What does this PR do?
Add pagination to articles

#### Description of Task to be completed?
- Users should be able to see the number of pages available
- Users should be able to go to any page by clicking on the page number

#### How should this be manually tested?
- clone this branch
- run `npm install`
- `npm run dev` to start the development server
- login to the app
- click on any of the categories to view the articles in that category
- if the category has more than 10 articles, you see the pagination icon beneath the articles. You can use it to traverse the pages

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[159987369]()

#### Screenshots (if appropriate)
**old**
<img width="1184" alt="screen shot 2018-11-19 at 4 34 25 pm" src="https://user-images.githubusercontent.com/26192691/48717106-078fc400-ec19-11e8-9eff-0e243e9d3b88.png">

**new**
![screen shot 2018-11-21 at 9 36 11 am](https://user-images.githubusercontent.com/26192691/48828815-070a4100-ed71-11e8-81bc-e3b694b72215.png)

#### Questions:
N/A
